package.json

```diff
@@ -113,6 +113,7 @@
     "symlink-dir": "^1.0.2",
     "typescript": "2.3.2",
     "typescript-definition-tester": "0.0.5",
+    "vue": "2.3.3",
```

Comment:
Both. After @gaspard PR is in Cerebral should be enough as it gonna be copied automatically.